### DISCUSSION TITLE
Plugin issue lead to server error django.db.utils.ProgrammingError'

### DISCUSSION THREAD

**Discussion by TECH-SWOCA** [netbox-community/netbox#13405] [Q&A] [answered]
Hello, I was attempting to configure netbox-dns-plugin in our installation and I ran into some file permission errors (some things had been sudo'd some hadn't, it was a mess). In the process of trying to resolve those issues I uninstalled the plugin, which since it created database tables, I went through the drop database tables instructions at the bottom (https://docs.netbox.dev/en/stable/plugins/). 

Now I've reinstalled the plugin and everything seems to be working properly except manage.py migrate won't recreate the tables I dropped. Which means when I attempt to navigate to any of the plugins pages I get an error similar to the one below

> <class 'django.db.utils.ProgrammingError'>
> 
> relation "netbox_dns_view" does not exist
> LINE 1: SELECT COUNT(*) AS "__count" FROM "netbox_dns_view"
>                                           ^
> 
> 
> Python version: 3.10.12
> NetBox version: 3.5.6

**Comment by abhi1693** [comment, marked as answer]
You also need to remove the migration from `django_migrations` table. You will have to look inside the table and figure which rows belong to the plugin, remove the said rows and then reattempt the installation.

**Reply by TECH-SWOCA** [reply]
Thank you thank you!

I could tell there must've been something somewhere flagging that the migration had already been completed but I didn't want to start digging through tables completely blind looking for such a thing.
